### PR TITLE
Moving styles around & cleaning up things

### PR DESCRIPTION
**Pull request by xavxyz**
Context
---
👋 This is preliminary PR to add an animated layout. It's kind of me saying hello to the codebase.

Changes
---
- Global layout styles are hardcoded in `index.html` so they are loaded before the JS bundle is.
- Pass Spotify's user `displayName` in the `GetUsername` module.
- `AppStyles` becomes `SharedCss`
- Remove some unused variables and clear some warnings

Step by step
---
- 29e30ed — styles: scope User styles to the User module

- abd758a — styles: remove unused shared style pageSubtitle

- 5e47f6d — styles: make global body styles loaded before the javascript bundle

- b690944 — styles: make shared css more discoverable (AppStyles -> SharedCss)

- 9d59b0e — username: use Spotify's displayName if available

### CODE REVIEW

File: spotify-app/src/index.html [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
-</head>
-<body>
-  <div id="root"></div>
-  <script src="Index.js"></script>
-  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
-</body>
+    <link
```
Comment (yukims19) [inline spotify-app/src/index.html]: Let's remove the white space lines here. I think maybe it's the prettier in JS mode causing the weird html code styling?

File: spotify-app/src/index.html [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
+      href="https://fonts.googleapis.com/css?family=Montserrat"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous"
+    />
+    <style>
```
Comment (yukims19) [inline spotify-app/src/index.html:10]: Is there any reason that you added the body style here instead of in the `SharedCss.re`?

**Comment by yukims19** [comment]
I left a few comments for the `index.html`. But other than that, it looks great to me!

**Review by yukims19 — APPROVED** [review]
I left a few comments for the index.html. But other than that, it looks great to me!

File: spotify-app/src/index.html (hunk 1 above)
Comment (xavxyz) [reply spotify-app/src/index.html]: Sure, it's prettier format on save!

File: spotify-app/src/index.html (hunk 2 above)
Comment (xavxyz) [reply spotify-app/src/index.html:10]: Indeed there is one, global layout styles are hardcoded in index.html so they are loaded before the JS bundle is: global layout styles are not depending on the JS. 

To go farther, you could have a noscript tag there teling, hey you need js, still with styles applied

**Comment by xavxyz** [comment]
Updated code based on your comments and merged, 🎩

**Pull request merged**